.github/ISSUE_TEMPLATE/change-author-list.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Change the list of authors
+about: Add, remove, or reorder the authors on a paper
+title: Change authors on Anthology ID XXX
+labels: correction, author-list
+assignees: xinru1414
+---
+
+## Important
+
+* **Please be sure to add the Anthology ID in the title**
+* [See here](https://www.aclweb.org/anthology/info/corrections/) to read about the types of corrections.
+
+## Instructions
+
+You need permission from the relevant conference program chairs if you want to change the author list in the metadata.
+See [the notes on author changes](https://www.aclweb.org/anthology/info/corrections/#note-on-author-changes) for more information.
+This applies whether or not the PDF needs to be changed.
```

Comment:
I would change it to "**This also applies if the PDF does not need to be changed**"  -- we really need to drive that point home.